SPECS/nano/nano.spec

```diff
@@ -1,13 +1,15 @@
 Summary:        Text editor
 Name:           nano
 Version:        6.4
-Release:        1%{?dist}
+Release:        2%{?dist}
 License:        GPLv3+
 Vendor:         Microsoft Corporation
 Distribution:   Azure Linux
 Group:          Applications/Editors
 URL:            https://www.nano-editor.org/
 Source0:        http://www.nano-editor.org/dist/v6/%{name}-%{version}.tar.xz
+Patch0:		CVE-2024-5742.patch
```

Comment:
The %setup needs to be changed to %autosetup.

Reply:
Switched it to `%autosetup -p1`